src/org/usfirst/frc4904/sovereignty/FusedSensor.java

```diff
@@ -0,0 +1,50 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+public class FusedSensor<T> implements Fusible<T> {
+	protected final Fusible<T> primarySensor;
+	protected final Fusible<T> fallbackSensor;
+	protected final Fusible<T> lastResortSensor;
+
+	public FusedSensor(Fusible<T> primarySensor, Fusible<T> fallbackSensor, T defaultValue) {
+		this.primarySensor = primarySensor;
+		this.fallbackSensor = fallbackSensor;
+		this.lastResortSensor = new Fusible<T>() {
+			@Override
+			public T getValue() {
+				return defaultValue;
+			}
+
+			@Override
+			public boolean trustable() {
+				return false;
+			}
+		};
+	}
+
+	public FusedSensor(Fusible<T> primarySensor, Fusible<T> fallbackSensor) {
+		this(primarySensor, fallbackSensor, null);
+	}
+
+	protected Fusible<T> getActiveSensor() {
+		if (primarySensor.trustable()) {
+			return primarySensor;
+		} else if (fallbackSensor.trustable()) {
+			return fallbackSensor;
+		}
+		return lastResortSensor;
```

Comment:
Alternative? I was using it for its property of being the antonym of safely. In this case, since we are overriding the getValue method it can't throw an error, but an error should undoubtedly be thrown here as you commented so to resolve that I went with the dangerous route.